tests/functional/object/mcg/test_bucket_policy.py

```diff
@@ -324,7 +324,10 @@ def test_object_actions(self, mcg_obj, bucket_factory):
         logger.info(
             f"Adding object on bucket: {obc_obj.bucket_name} using user: {obc_obj.obc_account}"
         )
-        assert s3_put_object(
+        retry_s3_put_object = retry(boto3exception.ClientError, tries=4, delay=15)(
```

Comment:
@ypersky1980 @sagihirshfeld @udaysk23 would it make sense to add this retry in the `s3_put_object function` directly?

Reply:
@keesturam you are right, fixed this.